src/main/java/util/FileCopier.java

```diff
@@ -0,0 +1,18 @@
+package util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+/**
+ * Util class to move files from one place to another
+ *
+ * @author Laura Schmidt
+ * @version 1.0 Initial implementation
+ */
+public class FileCopier {
+    public void copyFile(String sourcePath, String targetPath) throws IOException {
+        Files.copy(Path.of(sourcePath), Path.of(targetPath), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Sollte hier vielleicht lieber ein non-replace-exisiting genutzt werden? Egal welche Variante wir nutzen, wir wollen ja eigentlich nie überschreiben. Entweder ist es eh die gleiche Datei von uns (dann is egal) oder der Nutzer hat sie bearbeitet (dann auf keinen Fall überschreiben)